src/Makefile

```
@@ -383,6 +383,21 @@ else
 	MAYBE_UNINSTALL_REDIS_SYMLINK=
 endif
 
+# Determine whether to use fast_float as a library or not. 
+BUILD_FAST_FLOAT:=yes
+ifeq ($(BUILD_FAST_FLOAT),yes)
+	DEPENDENCY_TARGETS+= fast_float
+        FINAL_LIBS += ../deps/fast_float/fast_float_strtod.o
+	# Mac uses Clang and the standard c++ library is named libc++. For others, it is named libstdc++. 
```

```suggestion
	# Prefer to use libc++ when compiling with Clang. 
```
MacOS does use clang, but clang is also available on Linux.

---

And Linux is also available on Mac.

Mac != MacOS.

---

Fixed the comment, this check is intended to support Clang not just Mac.